pkg/resources/metrics.go

```diff
@@ -129,17 +129,17 @@ func CreateRedisConnectivityAlert(ctx context.Context, client k8sclient.Client,
 		"severity":    "critical",
 		"productName": cr.Labels["productName"],
 	}
-
+	sopUrl := alertRedisConnectionFailed
```

Comment:
How about removing the variable assignment `sopUrl`, and just placing the name of the SOP URL into the `reconcilePrometheusRule` function, and into the `sop_url` annotation directly?

It might be a good idea to also rename the variables to make them more descriptive, i.e `alertPostgresInstanceUnavailable` to `alertPostgresInstanceUnavailableSOPUrl`, or something similar, if doing the above. WDYT?

Reply:
:+1: